<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Preferences</class>
 <widget class="QDialog" name="Preferences">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>351</width>
    <height>232</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Atomical Preferences</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QDialogButtonBox" name="cancelOK">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>311</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="fullScreenOptions">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>311</width>
     <height>61</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="title">
    <string>Full Screen Options:</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QCheckBox" name="fullScreenMenubar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:0px;</string>
    </property>
    <property name="text">
     <string>Show Menu Bar</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="fullScreenControls">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:0px</string>
    </property>
    <property name="text">
     <string>Show Controls</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>301</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="generalOptions">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>311</width>
     <height>101</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="title">
    <string>General Options:</string>
   </property>
   <widget class="QPushButton" name="color">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>161</width>
      <height>27</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string> Background Color...</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QGraphicsView" name="backgroundColorGraphics">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>60</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background:red;border:2px solid black;</string>
    </property>
    <property name="interactive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="nativeDialogs">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Use Native Controls</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>301</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelOK</sender>
   <signal>accepted()</signal>
   <receiver>Preferences</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>201</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelOK</sender>
   <signal>rejected()</signal>
   <receiver>Preferences</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>201</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>color</sender>
   <signal>clicked()</signal>
   <receiver>Preferences</receiver>
   <slot>showColor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>238</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fullScreenControls</sender>
   <signal>stateChanged(int)</signal>
   <receiver>Preferences</receiver>
   <slot>fullScreenControls(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>175</x>
     <y>115</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fullScreenMenubar</sender>
   <signal>stateChanged(int)</signal>
   <receiver>Preferences</receiver>
   <slot>fullScreenMenubar(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>175</x>
     <y>115</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nativeDialogs</sender>
   <signal>stateChanged(int)</signal>
   <receiver>Preferences</receiver>
   <slot>nativeDialogs(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>175</x>
     <y>115</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
